server/pages/index.jade

```diff
@@ -1,135 +0,0 @@
-doctype html
-//
```

Comment:
Added it back. Was trickier than I hoped -- didn't manage to add an HTML comment, so it's now done via `dangerouslySetInnerHtml` in a `<script>` tag, contained in a `/*... */` comment.